tests/antares/services/api_services/test_study_api.py

```diff
@@ -250,3 +251,70 @@ def test_read_study_api(self):
             assert actual_study.name == expected_study.name
             assert actual_study.version == expected_study.version
             assert actual_study.service.study_id == expected_study.service.study_id
+
+    def test_create_variant_success(self):
+        variant_name = "variant_test"
+        with requests_mock.Mocker() as mocker:
+            base_url = "https://antares.com/api/v1"
+            url = f"{base_url}/studies/{self.study_id}/variants?name={variant_name}"
+            variant_id = "variant_id"
+            mocker.post(url, text=variant_id, status_code=201)
```

Comment:
You should use res.json() to get the study_id so you should mock json instead of text here